installer/dev/project_reunion_definitions.h

```diff
@@ -57,6 +98,30 @@
     #define PR_FRAMEWORK_ARM64_RCENTRY PR_FRAMEWORK_ARM64_RC     PR_FRAMEWORK_ARM64_RCTYPE                 PR_FRAMEWORK_ARM64_PATH
 #endif
 
+#define PR_MAIN_X86_LISTENTRY { PR_MAIN_X86_ID, PR_MAIN_X86_TYPE },
+#define PR_MAIN_X86_RCENTRY PR_MAIN_X86_RCID PR_MAIN_X86_RCTYPE PR_MAIN_X86_PATH
+
+#define PR_MAIN_X64_LISTENTRY  { PR_MAIN_X64_ID, PR_MAIN_X64_TYPE },
+#define PR_MAIN_X64_RCENTRY PR_MAIN_X64_RCID PR_MAIN_X64_RCTYPE PR_MAIN_X64_PATH
+
+// arm64 is included only on arm64 builds
+#if defined(_M_ARM64)
+    #define PR_MAIN_ARM64_LISTENTRY { PR_MAIN_ARM64_ID, PR_MAIN_ARM64_TYPE },
+    #define PR_MAIN_ARM64_RCENTRY PR_MAIN_ARM64_RC     PR_MAIN_ARM64_RCTYPE                 PR_MAIN_ARM64_PATH
```

Comment:
Weird spacing. Accidental indentation? Tabs? Leftover indentation from an earlier simpler day when it was nicely lined up w/o any conditional logic or like complexities?

Same comment applies throughout

Reply:
Yes leftover, fixed.